zc_common/remote_resource/models.py

```diff
@@ -51,3 +52,95 @@ def deconstruct(self):
         del kwargs['max_length']
 
         return name, path, args, kwargs
+
+
+class GenericRemoteForeignKey(object):
```

Comment:
This class is an almost exact copy of [Django's GenericForeignKey](https://github.com/django/django/blob/master/django/contrib/contenttypes/fields.py#L21). The testing in slots and orders should be sufficient.